woocommerce/woocommerce-gutenberg-products-block
# Increase CSS specificity for local pickup address

## What

Fixes #11752

Increases the CSS specificity for the `Local Pickup` styles from
```scss
.wc-block-checkout__pickup-options, [...] {
  [...]
}
``` 
to
```scss
.wc-block-checkout__pickup-options, [...] {
  .wc-block-components-local-pickup-rates-control { 
    [...]
  }
}
``` 

## Why



The CSS definitions of the `Local Pickup` address have been overwritten by generic CSS styles. 

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Go to `/wp-admin/admin.php?page=wc-settings&tab=shipping&section=pickup_location`.
2. Enable local pickup option.
3. Add a pickup location.
4. Create a test page and add the Checkout block to it.
5. Go to the frontend and add a product to the cart.
6. Go to the test page with the Checkout block.
7. Select the shipping method `Local Pickup`.
8. Verify that the local pickup address is left aligned.


## Screenshots or screencast



<table>
<tr>
<td valign="top">Before:
<br><br>
<img width="688" alt="Screenshot 2023-11-15 at 00 14 00" src="https://github.com/woocommerce/woocommerce-blocks/assets/3323310/9bd1992d-c9b0-4e87-91fa-9a0cba038336">
</td>
<td valign="top">After:
<br><br>
<img width="685" alt="Screenshot 2023-11-15 at 00 04 48" src="https://github.com/woocommerce/woocommerce-blocks/assets/3323310/0d7b497f-3139-4d5b-92be-31bffea9fbaf">
</td>
</tr>
</table>

### Cross-browser screenshot

- Left: Chrome
- Centre: Safari
- Right: Firefox

<img width="2560" alt="Screenshot 2023-11-15 at 12 08 49" src="https://github.com/woocommerce/woocommerce-blocks/assets/3323310/400b4d6c-85ea-40f3-aab2-d757e8c10f58">



## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Fix: Left align local pickup address.
